src/views/limited-partners.njk

```diff
@@ -0,0 +1,19 @@
+{% extends "layout.njk" %}
+
+{% block beforeContent %}
```

Comment:
don't need this block
the backlink already includes by default

Reply:
block removed